src/App.tsx

```diff
@@ -1,204 +1,91 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoError } from './components/TodoError/TodoError';
+import { Error } from './types/Error';
+import { Filter } from './types/Filter';
+
+const USER_ID = 12007;
+
+const preparedTodos = (todosList: Todo[], selectedFilter: Filter): Todo[] => {
+  let filteredTodos = [...todosList];
+
+  switch (selectedFilter) {
+    case Filter.Active:
+      filteredTodos = todosList.filter(todo => !todo.completed);
+      break;
+
+    case Filter.Completed:
+      filteredTodos = todosList.filter(todo => todo.completed);
+      break;
+    default:
+      break;
```

Comment:
Do not leave the `default` block empty, there you can throw an error about the wrong type of filter